file_store/src/file_sink.rs

```diff
@@ -40,8 +40,8 @@ fn transport_sink(transport: &mut Transport) -> &mut Sink {
 #[derive(Debug)]
 pub enum Message {
     Data(oneshot::Sender<Result>, Vec<u8>),
-    Flush(oneshot::Sender<Result>),
-    FetchManifest(oneshot::Sender<Result<Vec<String>>>),
+    Commit(oneshot::Sender<Result<Vec<String>>>),
```

Comment:
Maybe a `Vec<String>` wants to be it’s own type or at least a type Alia’s here for clarity?

Reply:
Address 7301c64